application/config/language.php

```diff
@@ -58,7 +64,7 @@
   |    FALSE
   |
  */
-$config['detect_language'] = FALSE;
+$config['detect_language'] = TRUE;
```

Comment:
For the moment, we are going to leave this set to FALSE, because we have to do some tests and I think it generates conflict with the languages that are not yet available. That is to say, if the language is available, it should show English by default, but I think that if you use a language that is not available or the browser detects a language that does not exist, it generates an error.